content/v2.0/reference/release-notes/influxdb-cloud.md

```diff
@@ -10,9 +10,34 @@ aliases:
   - /cloud/about/release-notes
 ---
 
+InfluxDB Cloud updates occur continually. Find a compilation of monthly updates below.
+
+## 2020-09-02
+
+## Features
+
+### Pricing updates and Azure region
+
+- Update [pricing vectors](/influxdb/v2.0/account-management/pricing-plans/#pricing-vect) to determine pricing by the total data out and query count.
+
+- Add [Microsoft Azure support](/influxdb/v2.0/reference/urls/#microsoft-azure) for the `eastus` and `westeurope` regions. Each region has a unique InfluxDB Cloud URL and API endpoint.
+
+### Flux updates
+
+ - Add time-weighted average [`timeWeightedAvg()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/timeweightedavg/).
+ - Update [`integral()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/integral/) with linear interpolation.
+ - Add Flux query statistics to better understand query performance.
```

Comment:
@gunnaraasen @bpalani how can folks access these Flux query statistics?

Reply:
https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/profiler/#use-the-query-profiler-to-output-statistics-about-query-execution